include/verilated_types.h

```diff
@@ -70,6 +71,67 @@ extern std::string VL_TO_STRING_W(int words, const WDataInP obj);
 #define VL_OUT(name, msb, lsb) IData name  ///< Declare output signal, 17-32 bits
 #define VL_OUTW(name, msb, lsb, words) VlWide<words> name  ///< Declare output signal, 65+ bits
 
+//===================================================================
+// Activity trigger vector
+
+template <std::size_t T_size>  //
+class VlTriggerVec final {
+private:
+    // MEMBERS
+    std::array<bool, T_size> m_flags;  // State of the assoc array
+    static_assert(sizeof(m_flags) >= sizeof(bool) * T_size, "We don't want a special array");
+
+public:
+    // CONSTRUCTOR
+
+    VlTriggerVec() { clear(); }
```

Comment:
```suggestion
    VlTriggerVec() { clear(); }
    ~VlTriggerVec() = default;
```

Always write destructor when write constructor.

Reply:
Ok, but curious if this is just style or there is more to it?

Reply:
Some lint tools complain, though I forget which. It's part of the rule of three, if you declare constructor or destructor or copy should do all three, though the copy one I don't declare usually in these sort of cases (that's perhaps mis-style).